data/allowlist.yaml

```diff
@@ -0,0 +1,103 @@
+# The allowlist has a per-site setup, so that links from SiteA do not have to be allowed from SiteB
+
+# Remember that any scalar strings with escaped characters must be single-quoted
+
+bedrock_base_config: &bedrock_base_config
+  allowed_outbound_url_regexes:
+    # === Broad, trusted cases ===
+    - '^(\/|#).*' # Relative URLs and in-page anchors are OK
+
+    # === Mozilla-run things ===
+    - '^http[s]{0,1}:\/\/[^\/]+.mozilla.org\/(.*)' # TODO: test this regex thoroughly
+    - '^http[s]{0,1}:\/\/[^\/]+.mozilla.com\/(.*)' # TODO: test this regex thoroughly
+    - '^http[s]{0,1}:\/\/[^\/]+.firefox.com\/(.*)' # TODO: test this regex thoroughly
```

Comment:
Isn't `?` the same as `{0,1}`, so the above could be `'^https?:...`?

Reply:
Yeah, indeed - I need to circle back and tune all these as I add in tests. Will admit that I'm nowhere near superhero levels of regex

Obligatory XKCD - https://xkcd.com/208/

![](https://imgs.xkcd.com/comics/regular_expressions.png)